modules/apps/object/object-api/bnd.bnd

```diff
@@ -33,6 +33,7 @@ Export-Package:\
 	com.liferay.object.model,\
 	com.liferay.object.model.listener,\
 	com.liferay.object.odata.filter.expression.field.predicate.provider,\
+	com.liferay.object.permissions.util,\
```

Comment:
So you won't need to export this if we change the ObjectEntryModelPermissionsUtil class to object.entry.folder.util folder